python/feathub/feature_views/sliding_feature_view.py

```diff
@@ -49,6 +49,8 @@ def __init__(
         name: str,
         source: Union[str, TableDescriptor],
         features: Sequence[Union[str, Feature]],
+        sliding_window_timestamp_field: Optional[str] = None,
```

Comment:
Since the timestamp generated by sliding window likely have different semantics from the timestamp field in the source table, should we give it a new name instead of re-using source table field name by default?

Since we actually generate a new field in the output, should we also document this field (e.g. its semantic meaning) similar to Flink?